src/multiplot.ts

```diff
@@ -343,15 +350,13 @@ export class Multiplot {
       }
     }
     if (e.key == " ") {
-      e.preventDefault();
```

Comment:
You'll probably need to pay attention to this when merging your branches back, as it has been reverted in previous patches